doc/docs/modules/ROOT/pages/quickstart.adoc

```diff
@@ -273,11 +276,11 @@ spark.read.format("org.neo4j.spark.DataSource")
 |"1"|"Jane Doe"
 |===
 
-As you can see, the struct returned by the query is made of strings
-that you can then be cast Spark's getters (ie: `getLong`).
+As you can see, the Struct returned by the query is made of strings
+that you can then be cast Spark's getters (i.e., `getLong`).
```

Comment:
I think, this sentence is confusing. It looks like wordy or something is missing